### PR TITLE
Fix -b and add --core-cycles

### PR DESCRIPTION
**Pull request by snim2**
This PR is intended to help automate the paper figures. 

The first commit fixes `-b` when used with changepoints and outliers.

The second commit adds a new switch `--core-cycles` which allows the user to specify which core cycle counts should be plotted.

### CODE REVIEW

File: bin/plot_krun_results [hunk 1]
```diff
@@ -138,7 +138,7 @@ def add_instr_data(data_dictionary, key, machine, instr_dir):
 def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
          xlimits, with_outliers, unique_outliers, changepoints, changepoint_means,
          median=False, tukey=False, inset_xlimit=None, one_page=False,
-         legend_off=True, cycles_ylimits=None):
+         legend_off=True, core_cycles=(0,1,2,3), cycles_ylimits=None):
```
Comment (vext01) [inline bin/plot_krun_results:141]: This assumes we have 5 cores. Is that what we want? Does it do the right thing on openbsd, with no core data?

Comment (vext01) [reply bin/plot_krun_results:141]: Should the default be `None` to mean "all cores"?

File: bin/plot_krun_results [hunk 2]
```diff
@@ -502,7 +502,8 @@ def draw_process_exec(grid_cell, data, cycles_data,
 def draw_page(is_interactive, executions, cycles_executions,
               instr_executions, titles, window_size, xlimits,
               outliers, unique, common, changepoints, changepoint_means, median,
-              tukey, inset_xlimit=100, legend_off=True, cycles_ylimits=None):
+              tukey, inset_xlimit=100, legend_off=True, core_cycles=(0,1,2,3),
```
Comment (vext01) [inline bin/plot_krun_results:507]: same here.

**Comment by vext01** [comment]
Looks OK apart from that minor question.

**Comment by snim2** [comment]
OK, will update

**Comment by snim2** [comment]
OK, ready to squash?

**Comment by vext01** [comment]
Looks good. If it's tested, go ahead.

**Comment by snim2** [comment]
Squashed.

**Review by vext01 — APPROVED** [review]

**Pull request merged**